data/pug/pug.txt

```diff
@@ -482,7 +482,7 @@ outfit "Pug Biodefenses"
 	thumbnail "outfit/pug biodefenses"
 	"capture defense" 250
 	"unplunderable" 1
-	description "This defense system is designed to release a suite of toxins that are lethal to human beings (and probably to some other species, as well) but which have no effect at all on the Pug."
+	description "This semi-organic defense system is designed to release a highly lethal bioengineered virus capable of rapidly infecting any foreign entities present on the ship that it is installed on while leaving the crew of the ship unharmed. The virus, when combined with the defense system itself, is highly accurate at automatically determining friend from foe. However, it is still recommended that any new crew be given a temporary vaccine before being allowed onboard."
```

Comment:
I feel like this technically doesn't resolve the issue; the issue is that the old description makes it sound like Pug shouldn't be effected by biodefenses. The last bit of the new description makes it sound to me like if two ships are both using biodefenses, neither ship's crew should be effected. So I'd just remove the mention of any possible countermeasures to the biodefenses.
```suggestion
	description "This semi-organic defense system is designed to release a highly lethal bioengineered virus capable of rapidly infecting any foreign entities present on the ship that it is installed on while leaving the crew of the ship unharmed. The virus, when combined with the defense system itself, is highly accurate at automatically determining friend from foe."
```